filament/backend/src/vulkan/platform/VulkanPlatform.cpp

```diff
@@ -620,6 +620,9 @@ struct VulkanPlatformPrivate {
     uint32_t mProtectedGraphicsQueueFamilyIndex = INVALID_VK_INDEX;
     uint32_t mProtectedGraphicsQueueIndex = INVALID_VK_INDEX;
     VkQueue mProtectedGraphicsQueue = VK_NULL_HANDLE;
+    bool mDebugMarkersSupported = false;
+    bool mDebugUtilsSupported = false;
+    bool mMultiviewSupported = false;
```

Comment:
I don't think these need to be in the impl.  You can just use the values directly from the shared context and store them in VulkanContext.